apps/web/pages/settings/organizations/[id]/add-teams.tsx

```diff
@@ -25,7 +25,7 @@ AddNewTeamsPage.getLayout = (page: React.ReactElement, router: NextRouter) => (
       currentStep={5}
       maxSteps={5}
       isOptionalCallback={() => {
-        router.push(`/getting-started`);
+        router.push(`/event-types`);
```

Comment:
After creating teams in an ORG you need to complete on boarding for yourself as youre a new user right? 

This will create a flicker of the wrong page until our useRedirectToOnboarding hook fires in shell